services/server/src/sourcify-chains.ts

```diff
@@ -60,7 +62,7 @@ else {
 }
 
 // chains.json from ethereum-lists (chainId.network/chains.json)
-const allChains = chainsRaw as Chain[];
+const allChains = [...chainsRaw, ...extraChainsRaw] as Chain[];
```

Comment:
In the case when tempo is added, does `extraChainsRaw` override `chainsRaw` here? If so we should switch the ordering, or find another way to merge that will make sure the original chains.json values override extraChains.